tsconfig.json

```diff
@@ -23,6 +23,7 @@
     "esModuleInterop": true,
     // Babel builds source, Typescript is used only for .d.ts files
     "incremental": true,
-    "preserveWatchOutput": true
+    "preserveWatchOutput": true,
+    "erasableSyntaxOnly": true
```

Comment:
Just to check this does not alter the output it just enforces that the syntax meets the erasable syntax specs?

Reply:
The output will just be changed for the code that we changed. There will be no implicit change in the compiled JS.